Common/src/grid_movement/CFreeFormDefBox.cpp

```diff
@@ -973,17 +977,37 @@ bool CFreeFormDefBox::GetPointFFD(CGeometry *geometry, CConfig *config, unsigned
   unsigned short iVar, jVar, iDim;
   su2double X_0, Y_0, Z_0, Xbar, Ybar, Zbar;
 
-  bool Inside = false;
+  bool Inside = true;
   bool cylindrical = (config->GetFFD_CoordSystem() == CYLINDRICAL);
   bool spherical = (config->GetFFD_CoordSystem() == SPHERICAL);
   bool polar = (config->GetFFD_CoordSystem() == POLAR);
 
-  unsigned short Index[5][7] = {
-    {0, 1, 2, 5, 0, 1, 2},
-    {0, 2, 7, 5, 0, 2, 7},
-    {0, 2, 3, 7, 0, 2, 3},
-    {0, 5, 7, 4, 0, 5, 7},
-    {2, 7, 5, 6, 2, 7, 5}};
+  
+  // subdivide the box into separate tetrahedrons
+  // unsigned short Index[5][7] = {
+  //  {0, 1, 2, 5, 0, 1, 2},
+  //  {0, 2, 7, 5, 0, 2, 7},
+  //  {0, 2, 3, 7, 0, 2, 3},
+  //  {0, 5, 7, 4, 0, 5, 7},
+  //  {2, 7, 5, 6, 2, 7, 5}};
+
+  // indices of the FFD box. Note that the front face is labelled 0,1,2,3 and the back face is 4,5,6,7
+  unsigned short Index[6][5] = {
+  {0,1,2,3,0},  // front side
+  {1,5,6,2,1},  // right side
+  {2,6,7,3,2},  // top side
+  {3,7,4,0,3},  // left side
+  {4,5,1,0,4},  // bottom side
+  {4,7,6,5,4}}; // back side
```

Comment:
Move to a different PR please.
I know it is not practical when we need the fixes to continue with the day job, but it's how PR's should work and it is what we ask in the "contributing" guidelines.